microservices/realtime/configs/config.yml

```diff
@@ -0,0 +1,20 @@
+ws:
+   listen: 127.0.0.1:8081
+   cert: configs/server.crt
+   key: configs/server.key
+   jwt_public: /opt/irontec/ivozprovider/storage/jwt/public.pem
+   jwt_private: /opt/irontec/ivozprovider/storage/jwt/private.pem
+redis:
+   master: mymaster
+   sentinel_host: 127.0.0.1
+   sentinel_port: 26379
+   pool_size: 52
+   db: 1
+sentinels:
+   host: cache.ivozprovider.local
+   port: '26379'
```

Comment:
Are these diferent from sentinel_port and sentinel_host?

Reply:
Looks like this sentinel section of the config is obsolete. The config package in the code reads the `redis.xxx` values